Temmietayo/All-Recipes
# #169997299 Template Platform to Add a recipe

#### What does this PR do?
Add the template for user to add a recipe of their choice .

#### Description of Task to be completed?
Authenticated User can add recipe

#### How should this be manually tested?

- clone the repo

- cd into the template directory

- open the addrecipe.html with any browser of your choice

- these template should open in that browser
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[169997299](https://www.pivotaltracker.com/story/show/169997299)
